lib/gds_api/exceptions.rb

```diff
@@ -53,8 +53,8 @@ class HTTPUnavailable < HTTPIntermittentServerError; end
   class HTTPGatewayTimeout < HTTPIntermittentServerError; end
 
   module ExceptionHandling
-    def build_specific_http_error(error, url, details = nil, request_body = nil)
-      message = "URL: #{url}\nResponse body:\n#{error.http_body}\n\nRequest body:\n#{request_body}"
+    def build_specific_http_error(error, url, details = nil)
```

Comment:
Does this change mean we can remove `request_body` from `initialise()` too? 
https://github.com/alphagov/gds-api-adapters/blob/119ccfcb4659ee7afda1664923f8caca1d219700/lib/gds_api/exceptions.rb#L25

Reply:
Yeah it looks like it, good spot. Actually, it looks like we're never passing that argument from anywhere in the codebase, and we never noticed, because the argument value defaults to `nil`.

Reply:
Just noticed that if we do ever want request body info in Sentry, we could pass it to this `raven_context` method in `BaseError` https://github.com/alphagov/gds-api-adapters/blob/119ccfcb4659ee7afda1664923f8caca1d219700/lib/gds_api/exceptions.rb#L6, and make full use of Sentry's key filtering.